src/convertToObject.js

```diff
@@ -6,7 +6,15 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const objStyles = {};
+
+  sourceString.split(';').map((style) => {
+    const arrStyle = style.split(':').map((el) => el.trim());
+
+    objStyles[arrStyle[0]] = arrStyle[1];
```

Comment:
There is a potential issue if the `style` string does not contain a colon, as `arrStyle[1]` will be `undefined`. It would be safer to check if `arrStyle` has at least two elements before assigning them to the `objStyles` object.